### PR TITLE
Handle users that have blocked video autoplay

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Detect if videos are blocked from autoplaying in Chrome, Safari, and Firefox.

Hide the confirmation page video if the video is blocked from autoplaying.

## Justify why they are needed

We want the users to see the purchase summary.

Source: https://stackoverflow.com/questions/49939436/how-to-detect-if-chrome-safari-firefox-prevented-autoplay-for-video

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1662** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1662" target="_blank">Graphite</a> 
    * **PR #1664** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1664" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1664?utm_source=stack-comment).

File: apps/store/src/components/ConfirmationPage/SuccessAnimation.tsx [hunk 1]
```diff
@@ -11,20 +12,33 @@ type Props = {
...
         <VideoWrapper exit={{ opacity: 0 }}>
           <StyledVideo
-            autoPlay={true}
+            ref={videoRef}
             muted={true}
             playsInline={true}
             preload="auto"
             onEnded={handleEnded}
             poster={SUCCESS_ANIMATION_URL.replace('.mp4', '.jpg')}
+            onLoad={playVideo}
```
Comment (alebedev) [inline apps/store/src/components/ConfirmationPage/SuccessAnimation.tsx:41]: Is this the right event?  Let's check that it does not wait for full video before playing

We may need something like `onCanPlay` instead

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#events

**Review by alebedev — APPROVED** [review]
Looks good

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Job ID: UpyWZtuiPQwZp0mwDjHi.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1664)

**Pull request merged**